docs/development/decision-records/2024-09-16-tx-target-node-directory-proposal/README.md

```diff
@@ -0,0 +1,30 @@
+# Proposal for a TargetNodeDirectory in Tractus-X
+
+## Decision
+
+A new service that will contain Connectors' URL's of each partner a member wants the offers from, acting as the **TargetNodeDirectory for the partner's FederatedCatalog**.
+
+
+## Rationale
+
+While considering new interventions in the Federated Catalog, this decision aims to set the TargetNodeDirectory.
+From the [documentation](https://eclipse-edc.github.io/docs/#/submodule/FederatedCatalog/docs/developer/architecture/federated-catalog.architecture)
```

Comment:
this link is not working, maybe it's better to point directly to the readme in the repo:
https://github.com/eclipse-edc/FederatedCatalog/blob/e733355c6991ff633ee009bd5f35ced61e941da6/docs/developer/architecture/federated-catalog.architecture.md

Reply:
unsure why, but when created this doc that link pointed to correct documentation (that was the same as the readme.md file) and now it does not.
changed in [here](https://github.com/eclipse-tractusx/tractusx-edc/pull/1556/commits/a9cdd06d866b53b2ad56fbd0c92f0a67af751ae2).

Reply:
that's because upstream we just switched to the new docs site (https://eclipse-edc.github.io), but FC documentation still needs to be moved